### PR TITLE
Update Node and Dependencies

### PR DESCRIPTION
**Pull request by puttpotsawee**
This project was created for some time, upon cloning the project, I was unable to start the project.
Moreover, we've got many automated pull request from the bot (see [this](https://github.com/rentspree/tracker/pull/21), [this](https://github.com/rentspree/tracker/pull/19), or [this](https://github.com/rentspree/tracker/pull/18) and much more)

This Pull request update
- Node version from `10.15.3` to `19.7.0`
- run `npm audit fix`
- fix Jest config

The test should be able to run after this pull request.

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-10.15.3
+19.7.0
```
Comment (PichayuthK) [inline .nvmrc:1]: I think we should tls version 20 (iron)

Comment (puttpotsawee) [reply .nvmrc]: Make a commit to change this

**Review by peeramit-rentspree — APPROVED** [review]
LGTM

**Review by BankNatthapon — APPROVED** [review]
LGTM Too

**Pull request merged**